tests/screenshot/apps/components/PageViews.js

```diff
@@ -16,23 +16,39 @@ const PageComponents = [
 
 const BaseTests = [
 	{
-		component: <Panel><Header title="title of panel" subtitle="subtitle of panel" /><PageViews index={0}>{PageComponents[0]}</PageViews></Panel>,
+		component: <Panel><Header title="title of panel" subtitle="subtitle of panel" /><PageViews index={0} pageIndicatorPosition="top">{PageComponents[0]}</PageViews></Panel>,
 		wrapper: {full: true}
 	},
 	{
-		component: <Panel><Header title="title of panel" subtitle="subtitle of panel" /><PageViews index={0}>{PageComponents}</PageViews></Panel>,
+		component: <Panel><Header title="title of panel" subtitle="subtitle of panel" /><PageViews index={0} pageIndicatorPosition="top">{PageComponents}</PageViews></Panel>,
 		wrapper: {full: true}
 	},
 	{
-		component: <Panel><Header title="title of panel" subtitle="subtitle of panel" /><PageViews index={1}>{PageComponents}</PageViews></Panel>,
+		component: <Panel><Header title="title of panel" subtitle="subtitle of panel" /><PageViews index={1} pageIndicatorPosition="top">{PageComponents}</PageViews></Panel>,
 		wrapper: {full: true}
 	},
 	{
-		component: <Panel><Header title="title of panel" subtitle="subtitle of panel" /><PageViews index={2}>{PageComponents}</PageViews></Panel>,
+		component: <Panel><Header title="title of panel" subtitle="subtitle of panel" /><PageViews index={2} pageIndicatorPosition="top">{PageComponents}</PageViews></Panel>,
 		wrapper: {full: true}
 	},
 	{
-		component: <Panel><Header title="title of panel" subtitle="subtitle of panel" /><PageViews fullContents index={2}>{PageComponents}</PageViews></Panel>,
+		component: <Panel><Header title="title of panel" subtitle="subtitle of panel" /><PageViews fullContents index={2} pageIndicatorPosition="top">{PageComponents}</PageViews></Panel>,
+		wrapper: {full: true}
+	},
+	{
+		component: <Panel><Header title="title of panel" subtitle="subtitle of panel" /><PageViews index={0} pageIndicatorPosition="bottom">{PageComponents}</PageViews></Panel>,
```

Comment:
If `pageIndicatorPosition` has the default value `bottom`, can't we omit the prop?